src/actors/sheet-parts/health.hbs

```diff
@@ -1,43 +1,53 @@
+    <div class="flex-row nowrap flex-align-center">
+      <label>{{localize "RQG.Actor.Health.TotalHitPoints"}}
+        <input
+          type="number"
+          min="-99"
+          max="99"
+          name="system.attributes.hitPoints.value"
+          value="{{system.attributes.hitPoints.value}}"
+        > / {{system.attributes.hitPoints.max}}
+      </label>
+      <label class="text-right">{{localize "RQG.Actor.Health.HealingRatePerWeek" healingRate=system.attributes.healingRate }}</label>
+    </div>
+    {{#if embeddedItems.hitLocation}}
+    <div class="grid hit-location item-list">
+      <div class="headings"></div>
+      <div class="head1-4">{{localize "RQG.Actor.Health.HitLocation"}}</div>
+      <div class="head5"><span class="text-center">{{localize "RQG.Actor.Health.HitPointsAbbr"}}</span></div>
+      <div class="head6"><span class="text-right">{{localize "RQG.Actor.Health.ArmorPointsAbbr"}}</span></div>
+      <div class="head7">{{localize "RQG.Actor.Health.Wounds"}}</div>
+      {{#each embeddedItems.hitLocation}}
+        <div class="hit-location contextmenu item" data-item-id="{{id}}"><span class="text-right">{{system.dieFrom}}</span></div>
+        <div class="hit-location contextmenu item" data-item-id="{{id}}">-</div>
+        <div class="hit-location contextmenu item" data-item-id="{{id}}"><span class="text-right">{{system.dieTo}}</span></div>
+        <div class="hit-location contextmenu item" data-item-id="{{id}}">
+          {{name}}
+          {{#unless (eq system.hitLocationHealthState "healthy")}}
+            <i>&nbsp;{{toLowerCase (localize (concat "RQG.Item.HitLocation.HealthStatusEnum." system.hitLocationHealthState))}}</i>
+          {{/unless}}
         </div>
-        <div class="ml-5 text-right" title="{{localize "RQG.Actor.Health.AddWound"}}" data-item-add-wound><i class="fas fa-hand-holding-medical"></i></div>
-      </div>
-    {{/each}}
-  </div>
+        <div class="hit-location contextmenu item" data-item-id="{{id}}"><span class="text-right nowrap">{{system.hitPoints.value}} / {{system.hitPoints.max}}</span></div>
+        <div class="hit-location contextmenu item" data-item-id="{{id}}"><span class="text-right">{{system.armorPoints}}</span></div>
+        <div class="hit-location contextmenu item" data-item-id="{{id}}">
+          <div data-item-heal-wound title="{{localize "RQG.Actor.Health.HealWound"}}" class="text-right nowrap">
+            {{#each system.wounds}}
+            <i class="fas fa-notes-medical"></i> {{this}}
+            {{/each}}
+          </div>
+          <div class="ml-5 text-right" title="{{localize "RQG.Actor.Health.AddWound"}}" data-item-add-wound><i class="fas fa-hand-holding-medical"></i></div>
+        </div>
+      {{/each}}
+    </div>
+    {{else}}
+      <p>{{localize "RQG.Actor.Health.NoHitLocations"}}</p>
+    {{/if}}
+  {{else}}
+    <div class="flex-row nowrap flex-align-center">
+      <p>{{localize "RQG.Actor.Health.NoHitPoints"}}</p>
```

Comment:
If an actor doesn't have hitpoints or hitlocations then I think it should just be blank. It's not an error and I wouldn't want that text on a spirit for example. If an actor don't have any health related things to show I think the health section should just not be there (including the health header).

Reply:
I didn't mean it to be an "error" so much as an explanation in case you were looking for it.  But I can change it.

Reply:
Not that players are going be looking at spirits too much unless they have an associated spirit or fetch, but my players were definitely wondering where everything was when we started with just skills and hit locations.  I think they would have preferred to see the sections that were hidden.  So that was kind of my thinking.

Reply:
Done